siliconcompiler/remote/client.py

```diff
@@ -300,12 +302,19 @@ def remote_process(chip, steplist):
     # run remote process
     if chip.get('arg', 'step'):
         chip.error('Cannot pass "-step" parameter into remote flow.', fatal=True)
-    cur_steplist = chip.get('option', 'steplist')
-    pre_remote_steplist = {
-        'steplist': cur_steplist,
-        'set': chip.schema._is_set(chip.schema._search('option', 'steplist')),
+    cur_from_nodes = chip.get('option', 'from')
+    cur_to_nodes = chip.get('option', 'to')
+    pre_remote_from_nodes = {
+        'from_nodes': cur_from_nodes,
+        'set': chip.schema._is_set(chip.schema._search('option', 'from')),
+    }
+    pre_remote_to_nodes = {
+        'to_nodes': cur_to_nodes,
+        'set': chip.schema._is_set(chip.schema._search('option', 'to')),
```

Comment:
fundamentally, this needs to be fixed when we re-work the server.

Reply:
Honestly I don't know if this actually needed.
There was some logic like that too in siliconcompiler itself for the step/indexlist...

Reply:
I'll try if I can get it to work without.